ChildMindInstitute/mindlogger-admin-OLD-Vue
# Fix reviewer invites #323

Maps MRN to Profile ID in reviewer invites.

When a reviewer invite is created, the list of users in the "users" field is mapped from MRN to the corresponding profile IDs. If profile IDs are used instead, they won't be modified.
    
Issue #323

**Testing this PR:**
1. Log into the admin panel with an owner account.
2. Go to the users page of an applet
3. Make sure that you see the MRN in the Active Users table
4. Invite a reviewer using the Invitation Form using MRNs for the "users" field
5. Accept the invitation
6. Log into the admin panel with the reviewer account
7. Check whether you got access to the right users
